Zscaler/zscaler/block_ioc.py

```diff
@@ -10,24 +10,27 @@
 class ZscalerAction(Action):
     def zia_auth(self):
         try:
-            api = ZiaTalker(self.module.configuration["base_url"])
-            api.authenticate(
+            api = ZiaTalker(
+                cloud_name=self.module.configuration["base_url"],
                 api_key=self.module.configuration["api_key"],
                 username=self.module.configuration["username"],
                 password=self.module.configuration["password"],
             )
             return api
+
         except Exception as e:
-            print(f"ZIA authentication failed: {str(e)}")
+            self.log(f"ZIA authentication failed", level="critical")
+            self.log_exception(e)
             return None
```

Comment:
Instead of return None (that generated the error in the issue), I suggest raising the caught exception.